LBH.AdultSocialCare.Api/V1/Boundary/Common/Request/CarePackageDetailRequest.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using LBH.AdultSocialCare.Api.Attributes;
+using LBH.AdultSocialCare.Api.V1.AppConstants.Enums;
+using LBH.AdultSocialCare.Api.V1.Domain.Common;
+
+namespace LBH.AdultSocialCare.Api.V1.Boundary.Common.Request
+{
+    [GenerateMappingFor(typeof(CarePackageDetailDomain))]
+    public class CarePackageDetailRequest
+    {
+        // TODO: VK: extend validation to deny out of range enum values, add custom Newtonsoft converter
+        [Range(1, 3)]
```

Comment:
What do you think EnumDataType attribute Vlad? 

Can we use this attribute?         `[EnumDataType(typeof(PackageDetailType))]`  
Ref: https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.enumdatatypeattribute?view=netcore-3.1

Reply:
I was thinking about some ActionFilter  or any other extension point in validation, so we could validate enum range in one place for all enums without extra efforts.

If this won't work, we will use some attribute, more specific than Range

Reply:
That's make more sense.